projects/storefrontlib/src/cms-components/navigation/footer-navigation/scroll-to-top/scroll-to-top.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { ScrollToTopComponent } from './scroll-to-top.component';
+
+describe('ScrollToTopComponent', () => {
```

Comment:
Please write tests which check if the scrolling is happening, etc. Currently this is far too few tests.

Reply:
Ok